.gitignore

```diff
@@ -23,3 +23,4 @@ git.properties.json
 /pacts/
 /mochawesome-report
 /config/dev.yaml
+/functional-output
```

Comment:
Why did you need to add this?

Reply:
if you run the end to end test, it produced the functional-output, and so i put it inside gitignore so it didnt keep showing up when i tried to do commit. (it's not related to this ticket, but i think it;s good to put it inside gitignore)